### PR TITLE
Enable ad refresh for A-Frame and Wonderland Engine

### PR DESCRIPTION
**Pull request by msub2**
This implements ad refreshing if the banner is in view, checking every 30 seconds. It also fixes a scaling issue on A-Frame with prebid banners that do not have absolute dimensions.

### CODE REVIEW

File: aframe/src/index.js [hunk 1]
```diff
@@ -63,16 +65,21 @@ async function createBanner(el, adUnit, format, style, height, beacon) {
...
+      return banner;
+    });
+  
     bannerPromise.then(banner => updateBanner(banner, plane, el, adUnit, format, style, height, beacon));
-  //}, 5000);
+  }
+
+  getBanner();
+  if (isBeta) {
+    setInterval(getBanner, 30000);
```
Comment (limbofeather) [inline aframe/src/index.js:138]: Would be helpful to have this number be a descriptive constant so we know what the 30000 means

File: wonderland/src/index.js [hunk 2]
```diff
@@ -95,14 +97,23 @@ export class ZestyBanner extends Component {
         .then(value => {
           this.zestyNetworking = Object.assign({}, value);
           this.startLoading();
+          if (isBeta) {
+            setInterval(this.startLoading.bind(this), 30000);
```
Comment (limbofeather) [inline wonderland/src/index.js:102]: Can the `30000` be a descriptive constant here

**Review by limbofeather — APPROVED** [review]
Small nit but otherwise LGTM, a short description for the PR would be nice to give more context if there's anything that you want there.

File: wonderland/src/index.js (hunk 2 above)
Comment (msub2) [reply wonderland/src/index.js]: Done

File: aframe/src/index.js (hunk 1 above)
Comment (msub2) [reply aframe/src/index.js]: Done

**Pull request merged**